doc/developer-guides/hld/atkbdc-virt-hld.rst

```diff
@@ -0,0 +1,64 @@
+.. _atkbdc_virt_hld:
+
+atkbdc emulation
+################
+
+This document describes atkbdc virtualization implementation in ACRN device model. Atkbdc device contains ps2 keyboard and mouse.
```

Comment:
```suggestion
This document describes ``atkbdc`` virtualization implementation in ACRN device model. Atkbdc device contains ps2 keyboard and mouse.
```

Reply:
More than "contains", wouldn' it more appropriate to say "emulates a PS/2 keyboard and mouse"?